tests/acceptance/signup-flow-test.js

```diff
@@ -0,0 +1,50 @@
+import { module, test } from 'qunit';
+import { visit, currentURL, fillIn } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+
+module('Acceptance | signup flow', function(hooks) {
+
+  setupApplicationTest(hooks);
+
+  test('http://localhost:4200/signup', async function(assert) {
```

Comment:
I think a more descriptive name test name will be good instead of `http://localhost:4200/signup`